R/item_append_files.R

```diff
@@ -14,17 +14,18 @@
 #'  item_append_file("54e265a4e4b08de9379b4dfb", '/foo/bar/baz.zip', sb)
 #' }
 #'@export
-item_append_file = function(id, filename, session){	
+item_append_files = function(id, files, session){	
```

Comment:
@dblodgett-usgs note this API change if Luke accepts this PR. Since this supports multi-file uploads, the name and param was changed.